src/components/system/SystemAlert.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from "react";
+import { connect } from 'react-redux';
+import PropTypes from "prop-types";
+import * as R from 'ramda';
+import moment from 'moment';
+import { differenceInMinutes, differenceInHours } from 'date-fns';
+import systemActions from '../../features/system/actions';
+import '../../../assets/css/widgets.css';
+
+class Alert extends Component {
+
+  componentDidMount() {
+    this.getSystemInfo();
+  }
+
+  componentDidUpdate() {
+    this.getSystemInfo();
+  }
+
+  getSystemInfo() {
+    const { session, dispatch } = this.props;
+
+    if (!R.isEmpty(session) && session.authenticated) {
+      dispatch(systemActions.systemPollStartAction());
+    } else {
+      dispatch(systemActions.systemPollStopAction());
+    }
+  }
+
+  renderAlertMessage() {
+    const { system } = this.props;
+    const deviceTime = new Date();
+    const serverTime = new Date(`${system.systemDate}T${system.systemTime}Z`);
+
+    const deviceTimeInUTC = deviceTime.toISOString();
+
+    const hasTimeZoneOffset = serverTime.getTimezoneOffset() === deviceTime.getTimezoneOffset();
```

Comment:
I think the issue here is that the system.systemDate and system.systemTime do not include the timezone, so I believe serverTime.getTimezoneOffset() will always return 0?  I will merge and test.

And shouldn't this be "!=="